rs/execution_environment/src/query_handler/tests.rs

```diff
@@ -269,185 +250,9 @@ fn query_calls_disabled_for_application_subnet() {
     }
 }
 
-#[test]
-fn legacy_query_callgraph_depth_is_enforced() {
```

Comment:
This is obsolete now that the legacy ICQC is removed, there's a version covering composite queries case.